components/LargeInfoSection.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @next/next/no-img-element */
+import React, { useEffect } from 'react';
+import { motion, useAnimation } from 'framer-motion';
```

Comment:
Can we do this animation without this lib using css?

Reply:
Now, I am using tailwind classes